situp-core/build.gradle

```diff
@@ -31,4 +35,8 @@ jar {
         configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
     }
 
-}
\ No newline at end of file
+    exclude 'META-INF/*.RSA'
+    exclude 'META-INF/*.SF'
+    exclude 'META-INF/*.DSA'
```

Comment:
I merged my PR, this should be a merge conflict.

Reply:
Sure, i'll look out for that